### PR TITLE
Dummy tiles

### PR DESCRIPTION
**Pull request by vasystus**

### CODE REVIEW

**Review by joe-dev-public — COMMENTED** [review]
Can we add the stars counter (but not the home button) to the top left of the child-landing page? :)

File: pages/child-landing.js [hunk 1]
```diff
@@ -6,28 +6,105 @@ import { useEffect } from "react";
...
+        <Link href="404" passHref>
+          <StoryTime>Story Time</StoryTime>
+        </Link>
+      </GameSelection>
+
     </>
   );
 }
 
+const OneHundred = styled.a`
```
Comment (PJSalter) [inline pages/child-landing.js:30]: Great styling, very clear, good use of rem ✨

File: pages/child-landing.js [hunk 2]
```diff
@@ -6,28 +6,105 @@ import { useEffect } from "react";
...
+    background-color: #cccccc;
+    background-image: linear-gradient(#11d600, #21a300, darkorange);
+  }
+`;
+
+const SayHi = styled.h2`
+  text-align: center;
+`;
+
+const GameSelection = styled.div`
```
Comment (PJSalter) [inline pages/child-landing.js:105]: Yaaaaaaay to the flexbox styling 🎨 🧑‍🎨

File: pages/child-landing.js [hunk 3]
```diff
@@ -6,28 +6,105 @@ import { useEffect } from "react";
 export default function ChildLanding({ learnerName }) {
   return (
     <>
-      <h2>{learnerName ? `Hi, ${learnerName}!` : "Hi there!"}</h2>
+      <SayHi>{learnerName ? `Hi, ${learnerName}!` : "Hi there!"}</SayHi>
```
Comment (PJSalter) [inline pages/child-landing.js:9]: The name of the component states what it does 😸 ✨

**Review by PJSalter — APPROVED** [review]

**Review by mariaalouisaa — APPROVED** [review]

**Pull request merged**